tests/src/Search/Web/ElasticTestCase.php

```diff
@@ -326,7 +326,7 @@ public function addDocumentsToElasticSearch(array $docs)
     public function createConfiguration()
     {
         if (file_exists($configFile = __DIR__.'/../../../../config/config.php')) {
-            $config = include __DIR__.'/../../../../config/config.php';
+            $config = include __DIR__.'/../../../../config.php';
```

Comment:
What's the `config/local.php` mentioned below?

Reply:
Obsolete, will remove all mentions.